pro/src/new_components/VenueForm/EACInformation/__specs__/EACInformation.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import '@testing-library/jest-dom'
+
+import { render, screen } from '@testing-library/react'
+import { createBrowserHistory } from 'history'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Router } from 'react-router-dom'
+
+import { IVenue } from 'core/Venue'
+import { configureTestStore } from 'store/testUtils'
+
+import EACInformation from '../EACInformation'
+
+const CREATION_TEXT = /Renseigner mes informations/
+const UPDATE_TEXT = /Modifier mes informations/
+const INFORMATION_BANNER_TEXT = /Une fois votre lieu créé/
+
+const renderEACInformation = async ({
+  venue,
+  isCreatingVenue,
+}: {
+  venue?: IVenue | null
+  isCreatingVenue: boolean
+}) => {
+  render(
+    <Router history={createBrowserHistory()}>
+      <Provider store={configureTestStore({})}>
+        <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />
+      </Provider>
+    </Router>
+  )
+}
+
+describe('components | EACInformation', () => {
+  it('should have the button disabled', async () => {
+    await renderEACInformation({
+      isCreatingVenue: true,
+      venue: null,
+    })
+
+    expect(screen.queryByText(CREATION_TEXT)).toHaveAttribute(
+      'aria-disabled',
+      'true'
+    )
+  })
+
+  it('should have the button with creation text with empty details', async () => {
+    const venue = {
+      id: 'V1',
+    } as unknown as IVenue
+
+    await renderEACInformation({
+      isCreatingVenue: false,
+      venue: venue,
+    })
+
+    expect(await screen.queryByText(CREATION_TEXT)).toBeInTheDocument()
+  })
+
+  it('should have the button with update text', async () => {
```

Comment:
`should be able to update venue` ?

Reply:
Pareil que plus haut. Je teste le contenu du bouton. Est-ce une variation graphique ?